src/frontend/src/utils/webAuthnErrorUtils.ts

```diff
@@ -1,15 +1,26 @@
 import { displayError } from "$src/components/displayError";
 import { I18n } from "../i18n";
-import copyJson from "./webAuthnErrorUtils.json";
+import cancelCopy from "./webAuthnErrorCancel.json";
```

Comment:
So far we've called the json object `copyJson` and had the copy file have the same name as the file importing it (modulo the extension). Do you think it makes sense to do the same here to stay consistent?

If the file becomes too big we can always split it later

Reply:
Done. But now we can no longer use the spread operator. At some point it would be nice to have nested copy. I.e.
```json
{
  "en": {
    "foo": {
      "title": "..."
    },
    "bar": {
      "title": "..."
    }
  }
}